front/src/modules/trainschedule/components/ManageTrainSchedule/TrainSettings.tsx

```diff
@@ -77,6 +78,8 @@ export default function TrainSettings() {
     setStartTime(startTimeFromStore.substring(0, 19));
   }, [nameFromStore, departureTimeFromStore, initialSpeedFromStore]);
 
+  const invalidName = nameIsInvalid(name);
```

Comment:
```suggestion
  const isInvalidName = nameIsInvalid(name);
```
Maybe this name would be better because this variable is a boolean

Reply:
Done